rusoto/credential/src/environment.rs

```diff
@@ -341,8 +321,8 @@ mod tests {
         );
     }
 
-    #[test]
-    fn alternative_prefix() {
+    #[tokio::test]
```

Comment:
Not blocking this review, but _maybe_ the starting point of a longer-term discussion: I'm not sure if `#[tokio::test]` should be used in Rusoto, as I expect some customers would want to use this with `async-std`.

Reply:
As nice as that would be, rusoto uses hyper, and hyper uses tokio. Would take a lot more work to be able to swap out backend's ala the runtime crate.

Reply:
I mean, I don't think this is an issue that should be resolved _right now_, especially since this usage is primarily in a test target. It doesn't introduce any public interfaces.

As for the swapping out executors in Hyper, the primary—and only?—client side impact in Hyper is DNS resolution, which consists of swapping out the `HttpConnector`. I can open an issue to discuss the impact of _that_ change.

No concerns, sorry!